src/sentry/integrations/slack/requests/base.py

```diff
@@ -115,20 +115,20 @@ def integration(self) -> RpcIntegration:
         return self._integration
 
     @property
-    def channel_id(self) -> str:
+    def channel_id(self) -> str | None:
         return get_field_id(self.data, "channel")
```

Comment:
could we use `_get_field_id_option` here as well, or no? Wonder why it's different